src/current/cockroachcloud/managed-backups.md

```diff
@@ -34,6 +34,10 @@ Refer to the [Cloud Console](#restore-a-basic-cluster) section for details on vi
 
 ### Standard and Advanced clusters
 
+{{site.data.alerts.callout_info}}
+Configurable managed backup settings are available in all [supported versions]({% link releases/release-support-policy.md %}) of CockroachDB on {{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters.
```

Comment:
Should the supported versions link include the `#supported-versions` anchor?

Reply:
Yes defo, thanks for that